gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -133,9 +133,6 @@ private static void configureErrorProneOptions(
                 "CanIgnoreReturnValueSuggester",
                 "InlineMeSuggester",
                 "PreferImmutableStreamExCollections",
-                // StringCaseLocaleUsage duplicates our existing DefaultLocale check which is already
-                // enforced in some places.
-                "StringCaseLocaleUsage",
```

Comment:
I see 5 references to `DefaultLocale` in our repos:
- 3 are increasing the severity to `WARNING` or `ERROR`, so this change is fine for those repos
- 2 are disabling this check, so those 2 repos will need to replace `DefaultLocale` with `StringCaseLocaleUsage`

Reply:
>  so this change is fine for those repos

I believe compilation will fail because no check exists with name `DefaultLocale` after this PR merges

Reply:
Here are the PRs for 2 of those 3. I'll also put up a PR for the one internal repo:
- https://github.com/palantir/atlasdb/pull/6605 (AtlasDB's baseline upgrade is [blocked](https://github.com/palantir/atlasdb/pull/6470) on other manual changes anyways)
- https://github.com/palantir/dialogue/pull/1946